wagtail/admin/templates/wagtailadmin/pages/listing/_locked_indicator.html

```diff
@@ -7,6 +7,11 @@
 {% endcomment %}
 
 {% if page.locked %}
-    {# TODO: Different look when it's locked by current user? #}
-    <span class="indicator locked-indicator icon icon-locked" title="{% trans 'This page is locked to further editing' %}"></span>
+    <span class="indicator locked-indicator {% if page.owner == request.user %}locked-indicator-owner{% endif %} icon icon-locked"
+    {% if page.owner == request.user %}
+        title="{% trans 'This page is locked to further editing by you' %}"
+    {% else %}
+        title="{% trans 'This page is locked to further editing' %}"
+    {% endif %}
+    ></span>
```

Comment:
Can we adopt the new icons here

```
{% icon ... %}
```

Will mean some styling updates but will lead to a better experience and consistent icon rendering